python/cxx/export.cpp

```diff
@@ -32,22 +32,10 @@ void python::common::export_all(py::module& module) {
 
 PYBIND11_MODULE(opmcommon_python, module) {
     python::common::export_all(module);
-}
-
-void python::common::export_all_opm_embedded(py::module& module) {
-    python::common::export_all(module);
-    module.attr("current_ecl_state") = std::make_shared<EclipseState>();
-    module.attr("current_summary_state") = std::make_shared<SummaryState>();
-    module.attr("current_schedule") = std::make_shared<Schedule>();
-    module.attr("current_report_step") = 0;
-    module.doc() = R"pbdoc(This is the opm_embedded module for embedding python code in PYACTION.)pbdoc";
-}
 
-/*
-  PYBIND11_MODULE create a Python of all the Python/C++ classes which are
-  generated in the python::common::export_all_opm_embedded() function in the wrapping code.
-  The same module is created as an embedded python module in PythonInterp.cpp
-*/
-PYBIND11_MODULE(opm_embedded, module) {
-    python::common::export_all_opm_embedded(module);
+    pybind11::module submodule = module.def_submodule("embedded");
+    submodule.attr("current_ecl_state") = std::make_shared<EclipseState>();
```

Comment:
what is the relationship between the objects instanced here and the simulator running the pyaction?

Reply:
The objects here are placeholders, they get set to the actual EclipseState, Schedule and SummaryState here:
https://github.com/OPM/opm-common/blob/master/opm/input/eclipse/Python/PyRunModule.cpp#L91

Reply:
Should we make a comment about this in the source?